src/librustc_codegen_llvm/common.rs

```diff
@@ -448,3 +448,101 @@ pub fn ty_fn_sig<'a, 'tcx>(cx: &CodegenCx<'a, 'tcx>,
         _ => bug!("unexpected type {:?} to ty_fn_sig", ty)
     }
 }
+
+pub fn size_and_align_of_dst<'a, 'tcx>(bx: &Builder<'a, 'tcx>, t: Ty<'tcx>, info: ValueRef)
```

Comment:
Why move this? I'd rather see `base` and `common` go away just as much as `glue`, so moving something here doesn't seem that useful. IMO this should be a method on `PlaceRef`, that's just `size_and_align_of`.